common/internal_model/src/test/scala/uk/ac/wellcome/models/work/generators/WorksGenerators.scala

```diff
@@ -200,11 +208,17 @@ trait WorksGenerators extends ItemsGenerators with ProductionEventGenerators {
 
   def createUnidentifiedInvisibleMetsWorkWith(
     sourceIdentifier: SourceIdentifier = createMetsSourceIdentifier,
-    items: List[Item[Unminted]] = List(createDigitalItem))
-    : UnidentifiedInvisibleWork =
+    items: List[Item[Unminted]] = List(createDigitalItem),
+    nImages: Int = 1): UnidentifiedInvisibleWork =
```

Comment:
maxImages?

Reply:
It creates exactly this number, to me `max` implies that there may be fewer...

Reply:
(`numImages` sounds more descriptive / cuter to me)